geth/transactions/queue/queue.go

```diff
@@ -8,17 +8,14 @@ import (
 
 	"github.com/ethereum/go-ethereum/accounts/keystore"
 	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/geth/account"
 	"github.com/status-im/status-go/geth/common"
 	"github.com/status-im/status-go/geth/log"
 )
 
 const (
 	// DefaultTxQueueCap defines how many items can be queued.
 	DefaultTxQueueCap = int(35)
```

Comment:
Interesting. Isn't `int` the default here and casting only needed when explicitly wanting something like `uint64` or similar.